DOCUMENTATION.md

```diff
@@ -1370,529 +1395,202 @@ The following are example configurations.
+```javascript
+Client.newHazelcastClient(clientConfig).then(function (client) {
+    // some operation
+});
+```
 
-Using mutual authentication, the clients also have their key stores and members have their trust stores so that the members can know which clients they can trust.
+**This client object is your gateway to access all the Hazelcast distributed objects.**
 
-To enable mutual authentication, firstly, you need to set the following property on the server side in the `hazelcast.xml` file:
+Let's create a map and populate it with some data, as shown below.
 
-```xml
-<network>
-    <ssl enabled="true">
-        <properties>
-            <property name="javax.net.ssl.mutualAuthentication">REQUIRED</property>
-        </properties>
-    </ssl>
-</network>
+```javascript
+var map;
+// Get the Distributed Map from Cluster.
+client.getMap('my-distributed-map').then(function (mp) {
+    map = mp;
+    // Standard Put and Get.
+    return map.put('key', 'value');
+}).then(function () {
+    return map.get('key');
+}).then(function (val) {
+    // Concurrent Map methods, optimistic updating
+    return map.putIfAbsent('somekey', 'somevalue');
+}).then(function () {
+    return map.replace('key', 'value', 'newvalue');
+});
 ```
 
-You can see the details of setting mutual authentication on the server side in the [Mutual Authentication section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#mutual-authentication) of the Hazelcast IMDG Reference Manual.
+As the final step, if you are done with your client, you can shut it down as shown below. This will release all the used resources and close connections to the cluster.
 
-At the Node.js client side, you need to supply an SSL `options` object to pass to
-[`tls.connect`](https://nodejs.org/api/tls.html#tls_tls_connect_options_callback) of Node.js.
+```javascript
+...
+.then(function () {
+    client.shutdown();
+});
+```
 
-There are two ways to provide this object to the client:
+## 7.2. Node.js Client Operation Modes
 
-1. Using the built-in `BasicSSLOptionsFactory` bundled with the client.
-2. Writing an `SSLOptionsFactory`.
+The client has two operation modes because of the distributed nature of the data and cluster: smart and unisocket.
 
-Below subsections describe each way.
+### 7.2.1. Smart Client
 
-**Using the Built-in `BasicSSLOptionsFactory`**
+In the smart mode, the clients connect to each cluster member. Since each data partition uses the well known and consistent hashing algorithm, each client can send an operation to the relevant cluster member, which increases the overall throughput and efficiency. Smart mode is the default mode.
 
-Hazelcast Node.js client includes a utility factory class that creates the necessary `options` object out of the supplied
-properties. All you need to do is to specify your factory as `BasicSSLOptionsFactory` and provide the following options:
+### 7.2.2. Unisocket Client
```

Comment:
nit: we should show how to configure unisocket mode here.

Reply:
Added a link to the section which describes how configure this setting to the parent section (7.2). Is it okay for you ?

Reply:
Sure, a link will work just fine.